engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-subscriptions-modal.html

```diff
@@ -7,14 +7,19 @@ <h4 data-block="modal-header" translate>
 
     <div class="row">
       <div class="col-sm-12">
-        <button type="button" class="btn btn-default" ng-click="autoAttach()" ng-disabled="table.numSelected > 0">
+        <button type="button" class="btn btn-default" ng-click="autoAttach()" ng-disabled="table.numSelected > 0 || contentAccessMode === 'org_environment'">
           <span translate>Auto-Attach</span>
         </button>
-        <p class="help-text" ng-show="table.numSelected === 0">
+
+        <p class="help-text" ng-show="table.numSelected === 0 && contentAccessMode != 'org_environment'">
           Auto-attach available subscriptions to all selected hosts.
         </p>
 
-        <p class="help-text" ng-show="table.numSelected > 0">
+        <p class="help-text" ng-show="contentAccessMode === 'org_environment'">
+          This organization has Simple Content Access enabled. Hosts can consume from all repositories in their Content View regardless of subscription status.
+        </p>
```

Comment:
prefer ```<div content-access-mode-banner/>``` over this

Reply:
Even that if part is covered-> ```<p class="help-text" ng-show="contentAccessMode === 'org_environment'">```
I'd suggest replacing lines 18-20 with just <div content-access-mode-banner/>

Reply:
@parthaa I've left the p tag on to provide a space between the button and the banner.